app/services/doc_auth/response.rb

```diff
@@ -41,6 +43,7 @@ def merge(other)
         pii_from_doc: pii_from_doc.merge(other.pii_from_doc),
         attention_with_barcode: attention_with_barcode? || other.attention_with_barcode?,
         doc_type_supported: doc_type_supported? || other.doc_type_supported?,
+        selfie_status: selfie_status,
         selfie_live: selfie_live?,
```

Comment:
Response to [this](https://github.com/18F/identity-idp/pull/10073#issuecomment-1942094218):
I think that confusingly we already have `selfie_status` tests in various specs. Search for "`xpect(response.selfie_status).to eq` to find a few.

Reply:
I could be wrong, but I _think_ what @dawei-nava  is saying in the linked comment is that `extra` hash in the [mock proofer](https://github.com/18F/identity-idp/blob/main/app/services/doc_auth/mock/result_response.rb#L21) should match the extra in the [real acuant response](https://github.com/18F/identity-idp/blob/main/app/services/doc_auth/acuant/responses/get_results_response.rb#L76).

But, I might not quite be understanding that right. From what I can see the `liveness_enabled` is included in [the Mock Proofer's response here](https://github.com/18F/identity-idp/blob/main/app/services/doc_auth/mock/result_response.rb#L245). But I could see an argument for adding the `selfie status`.

Maybe I'm way off base here though.

Reply:
Oh and while I was typing out my response I see Dawei replied! I'm still wrapping my head around it but I think maybe my comment was wrong, and I misunderstood.

Reply:
Yes, this following basically is the contract between the presenter and response(mock or trueid etc). The presenter expects it can fetch `@form_response.extra[:liveness_checking_required]`
```ruby
 def show_selfie_failures?
    @form_response.extra[:liveness_checking_required] == true
  end
```

Reply:
To clarify what I meant, I just pushed a check to the `result_response_spec.rb` to show it does contains the information the presenter expected.

Reply:
Thank you both for clarifying and thanks for adding that test Dawei.

> I think variable name liveness_enabled changed to liveness_checking_required
I'm not sure what happened, but I do think that I'm going to address it by changing `liveness_enabled` to `liveness_checking_required` in the mock proofer.